server/controllers/stock/reports/stock/value.js

```diff
@@ -97,12 +98,19 @@ async function reporting(_options, session) {
       item.weightedAverageUnitCost = weightedAverageUnitCost;
     });
 
+    stock.inventoryPrice = stock.inventory_price * rate;
+
     stock.stockQtyPurchased = quantityInStock;
     stock.stockUnitCost = weightedAverageUnitCost;
     stock.stockValue = (quantityInStock * weightedAverageUnitCost);
-    stockTotalValue += stock.stockValue;
+    stock.saleValue = (quantityInStock * stock.inventoryPrice);
 
-    stock.inventoryPrice = stock.inventory_price * rate;
+    // warn the user if the stock sales price is equivalent or _lower_ than the value of the stock
+    // the enterprise will not make money on this stock.
+    stock.hasWarning = stock.inventoryPrice <= weightedAverageUnitCost;
```

Comment:
It can be either - I'm open to changing it.  My thought is that the most common mistake users will make is setting the sale price = to the purchase price or vice versa, since they don't understand the difference between what they paid for the stock versus what they will sell it for.

Do you think we should limit it to `<`?

Reply:
That makes sense.  Just leave it as it is.